### ISSUE TITLE
Bot deletion confirmation old theme

### ISSUE THREAD

**Issue by kkrypt0nn** [top-gg/issues#840]
#### Expected Behavior

I expected to see the new UI and theme.

#### Observed Behavior

There is still the old UI and theme.

#### Applicable Links or Attachments

![image](https://user-images.githubusercontent.com/43011723/142729099-b3139041-4a80-4c1c-87cd-a230199ab46b.png)

#### Steps to Reproduce

1. Open top.gg
2. Go to one of your bots
3. Click on view the bot
4. Press the delete button

#### Web Browser and Version

Vivaldi - Version 94.0.4606.114 (Official Build) (64-bit)

#### Operating System and Version

Windows 11

#### Screen Resolution

3440x1440

**Comment by Hopefuls** [comment]
Yeah the deletion page was not brought to the new theme. Will definitely be changed soon.
